QWeb/internal/browser/chrome.py

```diff
@@ -102,11 +102,12 @@ def open_browser(executable_path: str = "chromedriver",
             options.add_argument("headless")
         if 'prefs' in kwargs:
             tmp_prefs = kwargs.get('prefs')
-            if isinstance(tmp_prefs, dict):
-                prefs = tmp_prefs
-            else:
-                prefs = util.prefs_to_dict(str(tmp_prefs).strip())
+            prefs = util.validate_prefs(tmp_prefs)
```

Comment:
Check review comment (change suggestion) to `util.validate_prefs()`

With that change this can be simplified to just
`prefs = util.prefs_to_dict(tmp_prefs)`

Reply:
not changed in this pr, so will make a specific refactor task about this

Reply:
was done due to linting actually. TBH can't see it being worth to be changed now. Maybe in later refactoring rounds.

Reply:
changed validate_prefs to parse_prefs